src/components/Select/components/SelectControl/SelectControl.scss

```diff
@@ -1,93 +1,27 @@
 @use '../../../../../styles/mixins';
 @use '../../../variables';
 
-@mixin button-text-color($color, $hoverColor: $color) {
-    &,
-    &:link,
-    &:visited,
-    &:active,
-    &:focus {
-        color: $color;
-    }
-
-    &:hover {
-        color: $hoverColor;
-    }
-}
-
 $block: '.#{variables.$ns-new}select-control';
+$buttonBlock: '.#{variables.$ns}button';
 
 #{$block} {
-    @include mixins.button-reset();
-    @include button-text-color(var(--yc-color-text-primary));
-    @include mixins.pin($block, $block, var(--yc-select-border-radius));
-
-    box-sizing: border-box;
-    display: inline-flex;
-    align-items: center;
     width: 100%;
     min-width: 34px;
```

Comment:
is it needed?

Reply:
`width: 100%` - to fill all available place of control`s wrapper
`min-width: 34px` - useless style